pkg/aws/aws.go

```diff
@@ -81,8 +82,15 @@ func GetDevpodVPC(ctx context.Context, provider *AwsProvider) (string, error) {
 	return "", nil
 }
 
-func GetDefaultAMI(ctx context.Context, cfg aws.Config) (string, error) {
+func GetDefaultAMI(ctx context.Context, cfg aws.Config, instanceType string) (string, error) {
 	svc := ec2.NewFromConfig(cfg)
+
+	architecture := "x86_64"
+	// Graviton instances terminate with g
+	if strings.LastIndex(strings.Split(instanceType, ".")[0], "g") != -1 {
```

Comment:
Instead of using strings.LastIndex != -1, please use strings.HasSuffix

Reply:
Done, thanks